main_comment.go

```diff
@@ -147,6 +151,46 @@ func processGitHubComment(
 	return nil
 }
 
+func protectBranch(conf *config, branchName string, pipelinePath string) error {
+	// https://docs.gitlab.com/ee/api/protected_branches.html#protect-repository-branches
+	opt := &gitlab.ProtectRepositoryBranchesOptions{
+		Name: &branchName,
```

Comment:
`allow_force_push` is `false` by default but I would add it here explicitly for clarity.